apps/widget/src/components/notification-center/NotificationCenterWidget.tsx

```diff
@@ -64,6 +64,8 @@ export function NotificationCenterWidget(props: INotificationCenterWidgetProps)
 
     window.addEventListener('message', handler);
 
+    window.parent.postMessage({ type: 'WIDGET_READY' }, '*');
```

Comment:
when the `Widget` is mounted - it first registers the event listener to the `message` event and then sends the event to `Embed` saying that it's ready to be initialized. The `window.parent` prop is a reference to the parent window or to self.